pkg/disttask/importinto/proto.go

```diff
@@ -158,19 +158,6 @@ func (t *importStepMinimalTask) String() string {
 	return fmt.Sprintf("chunk:%s:%d", t.Chunk.Path, t.Chunk.Offset)
 }
 
-// Chunk records the chunk information.
-type Chunk struct {
```

Comment:
why move it, lightning doesn't share this

Reply:
move it to resolve cycle import problem